server/routes/api/representatives.js

```diff
@@ -9,10 +9,11 @@ const router = express.Router()
 const { CICERO_API_KEY } = process.env
 
 const JURISDICTION_FILTER_MAP = {
-  federal: 'country',
-  state: 'administrativeArea1',
-  county: 'administrativeArea2',
-  municipality: 'locality'
+  federal: ['NATIONAL_UPPER', 'NATIONAL_LOWER'],
+  state: ['STATE_EXEC', 'STATE_UPPER', 'STATE_LOWER'],
+  county: ['COUNTY'],
+  local: ['LOCAL_EXEC','LOCAL'],
+  school: ['SCHOOL']
```

Comment:
I like this mapping!  I think this is the better of the two you mentioned in your PR description.  As you mentioned, the non-legislative jurisdictions are often empty.  I would consider removing the `school` designation and adding that plus `county` to their own `nonlegislative` category (see below).  That would make it a more broad designation that is more likely to return results and allow us to easily exclude it from the final app if we wanted, because often the people in these positions do not have direct voting power on the bills presented.

```suggestion
  federal: ['NATIONAL_UPPER', 'NATIONAL_LOWER'],
  state: ['STATE_EXEC', 'STATE_UPPER', 'STATE_LOWER'],
  local: ['LOCAL_EXEC','LOCAL'],
  nonlegislative: ['SCHOOL', 'COUNTY']
```

Reply:
@thyeggman Hey Jacob, I just want to make sure I understood your comment correctly. Do you mean when there's no filter, there should be no representatives shown? What I was assuming is all representatives for that zip code will be shown (regardless of jurisdiction) when there's no filter.

Reply:
> What I was assuming is all representatives for that zip code will be shown (regardless of jurisdiction) when there's no filter

Ah I see what the misunderstanding is.  When I'm talking about zip code, I'm also considering that a filter, whereas you were referring to the regional legislative filters.  Sorry for the confusion!  It looks like the current behavior already matches what I described.

As for this issue:

> If no filter is provided the default behavior I've coded is to not pass a district_type property into the API, therefore all officials (minus the ones we have filtered out manually i.e. President, Vice President) corresponding to that zip code are listed

@tdo95 I think it's fine if the shows the same officials multiple times.  We have some filtered out but it's important that people be able to look up every official for their jurisdiction.  If the list is too long, that's what the regional filters are for! 😄